manifests/secondarydns.yaml

```diff
@@ -95,6 +97,9 @@ spec:
         - containerPort: 5353
           name: dns
           protocol: UDP
+        - containerPort: 8081
```

Comment:
this is the coreDNS container, we just use coreDNS image for that
and created status-monitor in the same pod,
do we need probes for both containers?

Reply:
Yes, probes should be done for all the containers, at the end the coredns readiness is as important as the monitor-status one.